cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/hdfs/HdfsStorageClient.java

```diff
@@ -31,34 +30,15 @@ public class HdfsStorageClient implements StorageClient<FileSystem> {
   /** Initialize the HdfsStorageClient when the bean is accessed for the first time. */
   @PostConstruct
   public synchronized void init() throws IOException {
-    validateProperties();
+    log.info("Initializing storage client for type: " + HDFS_TYPE);
```

Comment:
Are these validations easily applicable to `LocalStorageClient` as well?

Reply:
Good point. I missed it. Part of it is.

Reply:
Actually it won't work for the local storage because the getEndpoint() and getRootpath() in local have special handling where they return the defaults instead of the one set explicitly in the storage properties.